conf/application.conf

```diff
@@ -80,11 +80,15 @@ microservice {
             merged-journey = true
         }
 
-      contact-frontend {
-        host = localhost
-        port = 9250
-        www = "http://localhost:9250"
-      }
+        bas-gateway-frontend {
+            sign-out-url = "http://localhost:9553/bas-gateway/sign-out-without-state"
+        }
```

Comment:
I'd suggest to move this link into `urls` section instead. This section `microservice.services` should contain other services' info, usually backend services. If you have a URL which you call from your view, for e.g., then `urls` is a better section to put it into.